### ISSUE TITLE
Признак наличия незакрытых вопросов по задаче в списке задач

### ISSUE THREAD

**Issue by milanse** [BlizD/Tasks#210]
Может стоит впилить такой реквизит ? Было бы удобно видеть задачи с открытыми вопросами ?

**Comment by d-hurricane** [comment]
Речь идет о наличии невыполненных пунктов списка комментариев/чеклиста?

**Comment by BlizD** [comment]
Отвечу на выходных

**Comment by milanse** [comment]
> Речь идет о наличии невыполненных пунктов списка комментариев/чеклиста?

Да. По сути список задач это мое рабочее место как руководителя постановщика и исполнителя задач.

Сейчас у меня редмайне настроено несколько представлений:

- Все открытые
- Мои задачи 
- На уточнении
- Решенные - требуется проверка

Далее сортирую по датам важности и исполнителям и вперед.

**Comment by d-hurricane** [comment]
Я сейчас работаю над оптимизацией заполнения списков Канбан-доски. Могу в рамках этой задачи добавить и нужный признак. Уточните только, чем не подходит вывод чек-листа дробью/процентом в третьей строке? Вам необходим именно булев признак? Или речь вовсе не о канбан-доске, а об основной форме списка задач?

**Comment by milanse** [comment]
> Я сейчас работаю над оптимизацией заполнения списков Канбан-доски. Могу в рамках этой задачи добавить и нужный признак. Уточните только, чем не подходит вывод чек-листа дробью/процентом в третьей строке? Вам необходим именно булев признак? Или речь вовсе не о канбан-доске, а об основной форме списка задач?

Об основном списке. В канбан тоже можно добавить, ну или статус автоматом считать по количеству открытых вопросов. 

Ещё в канбане я или не нашел или нет вывода количества задач в колонке, можно было бы сверху рядом с заголовком прикрутить.

**Comment by d-hurricane** [comment]
В основном списке такого поля не вижу. Предлагаю добавить соответствующее поле в регистр сведений "узСтатусыЗадач" (можно переименовать в "узСостоянияЗадач"). Поле будет вычисляться автоматом при записи задач.

В канбан-доске формально поле есть, но на форму не выведено. Т.к. используются таблицы значений, а не динамические списки, плюс элементы формы генерируются, то для отображения/скрытия полей приходится добавлять новые переключатели на закладку настроек. Универсально, наверное, не получится. Ну либо довольно трудоемко.
